content/docs/user-guide/managing-external-data.md

```diff
@@ -33,13 +34,17 @@ Currently, the following types (protocols) of external outputs (and
 - HDFS
 - Local files and directories outside the <abbr>workspace</abbr>
 
-> Note that these are a subset of the remote storage types supported by
+> Note that these are only a subset of the remote storage types supported by
 > `dvc remote`.
 
-> Avoid using the same [DVC remote](/doc/command-reference/remote) (used for
-> `dvc push`, `dvc pull`, etc.) for external outputs, because it may cause file
-> hash overlaps: the hash of an external output could collide with a hash
-> generated locally for another file with different content.
+> 💡 Note that external outputs are never pushed or pulled from/to
```

Comment:
Number of notes is getting out of hands. If we consider this important, let's improve the explanation how they work?

Reply:
I removed the first one (also from the external deps doc for consistency) and made the 2nd one not a block quote. So now there's just one block quote left at the bottom.